grpc/stream.go

```diff
@@ -127,7 +140,19 @@ func (s *stream) loop() {
 }
 
 func (s *stream) queueMessage(msg map[string]interface{}) {
+	time := time.Now()
+
 	s.tq.Queue(func() error {
+		metrics.PushIfNotDone(s.vu.Context(), s.vu.State().Samples, metrics.Sample{
+			TimeSeries: metrics.TimeSeries{
+				Metric: s.instanceMetrics.GRPCStreamsMessagesReceived,
+				Tags:   s.tagsAndMeta.Tags,
+			},
+			Time:     time,
+			Metadata: s.tagsAndMeta.Metadata,
+			Value:    1,
+		})
```

Comment:
nit; I would argue this should be outside of the event loop.

And if it isn't, shouldn't `time.Now` call be inside teh event loop as well ?

Reply:
right, I'll adjust :+1: